libvast/src/error.cpp

```diff
@@ -11,13 +11,20 @@
  * contained in the LICENSE file.                                             *
  ******************************************************************************/
 
-#include <sstream>
+#include "vast/error.hpp"
+
+#include "vast/detail/assert.hpp"
 
 #include <caf/pec.hpp>
 #include <caf/sec.hpp>
 
-#include "vast/detail/assert.hpp"
-#include "vast/error.hpp"
+#include <sstream>
+#include <string>
+
+// FIXME: workaround for actor-framework/actor-framework#940
+namespace caf {
+std::string to_string(pec x);
+}
```

Comment:
```suggestion
} // namespace caf
```

Would you mind adding concrete guidance on how to fix this in the future? e.g., "remove this forward declaration after upgrading to 17.3"

Reply:
I've addressed this.